src/services/avatars.ts

```diff
@@ -0,0 +1,166 @@
+     */
+    async getBrowser(code: string, width: number = 100, height: number = 100, quality: number = 100): Promise<string> {
+        let path = '/avatars/browsers/{code}'.replace(new RegExp('{code}', 'g'), code);
+        
+        return await this.client.call('get', path, {
+                    'content-type': 'application/json',
+               },
+               {
+                'width': width,
+                'height': height,
+                'quality': quality
+            });
+    }
+
+    /**
+     * Get Credit Card Icon
+     *
+     * Need to display your users with your billing method or their payment
+     * methods? The credit card endpoint will return you the icon of the credit
+     * card provider you need. Use width, height and quality arguments to change
+     * the output settings.
+     *
+     * @param string code
+     * @param number width
+     * @param number height
+     * @param number quality
+     * @throws Exception
+     * @return Promise<string>
+     */
+    async getCreditCard(code: string, width: number = 100, height: number = 100, quality: number = 100): Promise<string> {
+        let path = '/avatars/credit-cards/{code}'.replace(new RegExp('{code}', 'g'), code);
+        
+        return await this.client.call('get', path, {
+                    'content-type': 'application/json',
+               },
+               {
+                'width': width,
+                'height': height,
+                'quality': quality
+            });
+    }
+
+    /**
+     * Get Favicon
+     *
+     * Use this endpoint to fetch the favorite icon (AKA favicon) of a  any remote
+     * website URL.
+     *
+     * @param string url
+     * @throws Exception
+     * @return Promise<string>
+     */
+    async getFavicon(url: string): Promise<string> {
+        let path = '/avatars/favicon';
+        
+        return await this.client.call('get', path, {
+                    'content-type': 'application/json',
+               },
+               {
+                'url': url
+            });
+    }
+
+    /**
+     * Get Country Flag
+     *
+     * You can use this endpoint to show different country flags icons to your
+     * users. The code argument receives the 2 letter country code. Use width,
+     * height and quality arguments to change the output settings.
+     *
+     * @param string code
+     * @param number width
+     * @param number height
+     * @param number quality
+     * @throws Exception
+     * @return Promise<string>
+     */
+    async getFlag(code: string, width: number = 100, height: number = 100, quality: number = 100): Promise<string> {
+        let path = '/avatars/flags/{code}'.replace(new RegExp('{code}', 'g'), code);
+        
+        return await this.client.call('get', path, {
+                    'content-type': 'application/json',
+               },
+               {
+                'width': width,
+                'height': height,
+                'quality': quality
```

Comment:
You can use the Object Shorthand syntax and write it as:
```suggestion
                width,
                height,
                quality
```